tests/src/test/scala/system/basic/WskActionTests.scala

```diff
@@ -71,6 +71,21 @@ class WskActionTests extends TestHelpers with WskTestHelpers with JsHelpers with
     }
   }
 
+  it should "invoke an action that throws an uncaught exception and returns correct status code" in withAssetCleaner(
```

Comment:
System tests are intended to for sniffing the health of the system. This seems unnecessary here and the test could be moved to the core actions test suite, no?

Reply:
This is to test that during execution of the action the correct status is returned. I'm not sure how this could be tested without actually invoking the action.

Reply:
That’s fine. There are “basic” action tests in core that do the same. Historically the system package implied tests might be used to check the health of a deployment.